terraso_backend/apps/shared_data/views.py

```diff
@@ -52,6 +53,12 @@ def post(self, request, **kwargs):
                 context=ErrorContext(model="DataEntry", field="data_file"),
             )
             return JsonResponse({"errors": [{"message": [asdict(error_message)]}]}, status=400)
+        if not is_valid_shared_data_type(request.FILES.getlist("data_file")):
+            error_message = ErrorMessage(
+                code="Invalid media type",
```

Comment:
Issue: Change the code so that it can be potentially localized in the clients side
```suggestion
                code="invalid_media_type",
```